avocado/utils/gdb.py

```diff
@@ -59,11 +59,11 @@
 REMOTE_TRANSMISSION_FAILURE = '-'
 
 #: How the remote protocol flags the start of a packet
-REMOTE_PREFIX = '$'
+REMOTE_PREFIX = b'$'
```

Comment:
I was talking about the commit style because we have two commits with same title and descriptions one after another. IMO, this could be merged.

Reply:
Oh, one was about remote_*encode()* and the other was about remote_*decode()*. Sure, they could be merged, but they are about different methods.